app/assets/javascripts/components/nav_signals/active_signals.js.jsx

```diff
@@ -0,0 +1,37 @@
+var ActiveSignalPanel = React.createClass({
+  render: function() {
+    return (
+      <div>
+        {this.renderPanel()}
+      </div>
+    );
+  },
+
+  renderPanel: function() {
+    signals = [];
+    this.props.signals.forEach(function(s, idx) {
+      signals.push(
+        <div className={'panel signal-panel '+ s.signal_type}>
+          <SignalIcon path={'icons/' + s.signal_type} />
+          <div className='panel-header'>
+            {'# ' + s.name}
+          </div>
+          <div className='panel-body'>
+            Send your users a special offer every time they send a custom hashtag
+          </div>
+          <div className='panel-status'>
+            <div className='signal-status'>
+              <div className='circle active'>
+                <span className='status'>ACTIVE</span>
```

Comment:
Here, I need to be able to look at `signal.active?` to either 
1. show active or inactive in text.
2. switch btw circle class name from `active` to `inactive`

What would be the best way to do that?